### DISCUSSION TITLE
Nodes not synching...

### DISCUSSION THREAD

**Discussion by manokodu** [btcpayserver/btcpayserver#6192] [Technical Support]
I am getting this error for my donation button, not sure but assuming the store would have the same issue.

> "Pay Button request failed
> Warning: You have payment methods configured but none of them match any of the requested payment methods or the rate is not available. See logs below: 09/04/2024 13:45:51:Info Creation of invoice starting 09/04/2024 13:45:51:Info BTC_EUR: The rating rule is kraken(BTC_EUR) 09/04/2024 13:45:51:Info BTC_EUR: The evaluated rating rule is (51095.2, 51095.3) 09/04/2024 13:45:52:Info BTC (On-Chain): Payment method details creation took 0 milliseconds 09/04/2024 13:45:52:Error BTC: Payment method unavailable (Full node not available)"

Inside my account it says my nodes are synching, previously it wasn't having this issue. It is stuck at 98% now, and says this

> "Your nodes are synching …
> Your node is synching the entire blockchain and validating the consensus rules.
> 
> BTC
> NBXplorer headers height: 0
> The node is starting...
> [Watch this video](https://www.youtube.com/watch?v=OrYDehC-8TU) to understand the importance of blockchain synchronization. If you really don't want to sync and you are familiar with the command line, check [FastSync](https://docs.btcpayserver.org/Docker/fastsync/)."

I don't know what to do.

**Comment by forumsolutions79** [comment]
i am facing same issue any response on it

**Reply by manokodu** [reply]
No, therefore switching to another service.

**Comment by ilearnio** [comment]
Same issue. The API was working, now it doesn't. It returns errors such as 
```
{"code":"not-available","message":"BTC-CHAIN services are not currently available"}`)

{\"code\":\"generic-error\",\"message\":\"Error retrieving a matching payment method or rate.\\nBTC-CHAIN: Payment method unavailable (Full node not available)\\n\"}
```

The nodes stuck syncing and it broke the API as well as the payments (invoices API). Can't it at least not block things while syncing? Luckily I'm not in production, but still why it doesn't sync!

![image](https://github.com/user-attachments/assets/b3b5f5c7-7ee4-4932-9a7d-760e01600b64)

**Reply by pavlenex** [reply]
Update your btcpay https://x.com/btcpayserver/status/1922845019494318519?s=46
